include/usb/usb_device.h

```diff
@@ -166,16 +180,13 @@ struct usb_request_handlers {
 };
 
 /**
- * @brief USB device configuration
+ * @brief USB Class data
```

Comment:
I guess this and L185:L187 should be part of the subsequent patch.

Reply:
Yes. I will rebase it